doc/user_guide/z390/commands.md

```diff
@@ -78,24 +78,30 @@ The ez390 emulator supports the following interactive test commands when the
 * `addr=sdt` – set memory value  (i.e. 1r?=x'80' changes mem at (r1) 31 bit
 * `reg=sdt` - set register value (i.e. 15r=8 changes reg 15 to 8)
 * `A addr` – set or reset up to 100 instruction address stops with hex address or relative expression such as *+4
+* `AR nn` – display specified access register else all AR0-AR15
 * `B=addr` - set base for rel addr (ie B=15r% sets base to (r15) 24 bit
 * `D` – display DCB file information from TIOT
 * `E` – toggle between EBCDIC and ASCII mode
-* `F` – display floating point registers F0-FF
+* `F nn` – display floating point register else FPC and all F0-FF
```

Comment:
I find the reference F0-FF confusing as it seems to be hex numbers. I would like to clarify by changing F0-FF to FPR 0-F